python/lsst/daf/butler/core/fileTemplates.py

```diff
@@ -78,6 +87,33 @@ def __init__(self, config, default=None):
             else:
                 self.templates[key] = FileTemplate(templateStr)
 
+    def validateTemplate(self, *entities):
+        """Retrieve the template associated with each dataset type and
+        validate the dimensions against the template.
+
+        Parameters
+        ----------
+        *entities : `DatasetType`, `DatasetRef`, or `StorageClass`
+            Entities to validate against the templates.  True validation
+            requires that the supplied entities have access to dimension
+            graphs.
```

Comment:
Is "untrue validation" something that users should not do, because it will e.g. generate false positives, or something that might just not validate some things?

Reply:
I mean that if you validate a template against a StorageClass or DatasetType that does not have a universe then you can't really do much. I will rephrase.